### ISSUE TITLE
Parameter settings

### ISSUE THREAD

**Issue by ttkxyy** [kahnchana/svt#21]
Dear team,
Thank you again for your work on the code!
I have some doubts about the setting of SVT parameters, 
1. teacher_ temp the same as DINO, that is, teacher_ Temp=0.04 ?
2. In your paper, it was stated that `bs=32` for pre training on k400, with a learning rate of `lr=5.0e-4`. I would like to know if you have used **linear scaling rule**:` lr=5.0e-4 * batchsize/256`
